README.md

```diff
@@ -7,7 +7,7 @@
 [![License: Apache-2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/anchore/grype/blob/main/LICENSE)
 [![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/anchore/grype.svg)](https://github.com/anchore/grype)
 
-A vulnerability scanner for container images and filesystems. [Easily install the binary](#installation) to try it out.
+A vulnerability scanner for container images and filesystems. [Easily install the binary](#installation) to try it out. Works with [Syft](https://github.com/anchore/syft), the powerful SBOM tool for container images and filesystems.
```

Comment:
nit: it is common to spell out acronyms the first time they are introduced

Reply:
I think this makes it a bit wordy, TBH. But I'll try it out for now!